web/component2/src/demos/ref.dart

```diff
@@ -0,0 +1,338 @@
+// ignore_for_file: deprecated_member_use_from_same_package
+import 'dart:html';
+
+import 'package:over_react/over_react.dart';
+
+// ignore: uri_has_not_been_generated
+part 'ref.over_react.g.dart';
+
+// ------------ `uiForwardRef` with a function component (simple) ------------
```

Comment:
Sorry, I forgot to take a look at this file when I last reviewed.

Even the "simple" versions of these components are a bit complicated due to how many layers of components there are.

Would it be possible to update these demos to match the simplified versions in the `uiForwardRef` doc comment?

Reply:
Oh no problem! I was kinda surprised there wasn't a comment here ;) Yep, I can work on simplifying!

Reply:
Thanks! ❤️